src/views/screens/guided_search/date_questionnaire.njk

```diff
@@ -0,0 +1,45 @@
+{% extends "layout/template.njk" %}
+
+{% from "govuk/components/breadcrumbs/macro.njk" import govukBreadcrumbs %}
+{% from "govuk/components/date-input/macro.njk" import govukDateInput %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+
+{% block beforeContent %}
+  {{ super() }}
+  {{ govukBreadcrumbs({
+    items: [
+      {
+        text: "Home",
+        href: homePageUrl
+      },
+      {
+        text: "Questionnaire search",
+        href: "#"
+      }
+    ]
+  }) }}
+{% endblock %}
+
+{% block content %}
+<div class="govuk-grid-row">
+    <form method="POST" action="/guided-search" />
+      <div class="govuk-grid-column-two-thirds">
+        <h1 class="govuk-heading-m">
+          When was the data or information collected?
+        </h1>
+        <h2 class="govuk-heading-m">
```

Comment:
```
<h2 class="govuk-heading-m">
        Label
</h2>
{{ govukDateInput(fromDate) }}
```
A macro can be used to pass the necessary attributes and generate the HTML structure instead of having the same HTML structure multiple times.

Reply:
Can we pick this up later?